sdk/eventhubs/azure_messaging_eventhubs/README.md

```diff
@@ -296,7 +295,7 @@ Azure SDK for C++ is licensed under the [MIT](https://github.com/Azure/azure-sdk
 [source]: https://github.com/Azure/azure-sdk-for-rust/tree/feature/track2/sdk/eventhubs/azure_messaging_eventhubs
 [azure_identity_pkg]: https://docs.rs/azure_identity/latest/azure_identity/
 [default_azure_credential]: https://docs.rs/azure_identity/latest/azure_identity/struct.DefaultAzureCredential.html
-[cppdoc]: https://docs.rs/azure_messaging_eventhubs/latest/azure_messaging_eventhubs
-[cppdoc_examples]: https://github.com/Azure/azure-sdk-for-cpp/tree/main/sdk/eventhubs/azure-messaging-eventhubs/samples
+[rustdoc]: https://docs.rs/azure_messaging_eventhubs/latest/azure_messaging_eventhubs
+[rustdoc_examples]: https://github.com/Azure/azure-sdk-for-cpp/tree/main/sdk/eventhubs/azure-messaging-eventhubs/samples
```

Comment:
Once set up, the link checker will 404 and fail this. Probably better to use the `feature/track2` branch instead of `main`. We'll have a lot to flip later anyway.

Reply:
The rustdoc links aren't going to work until we actually publish the package event if we say feature/track2 branch.

And I think I do use feature/track2 in my link to the repo.

Reply:
The markdown-rendered links will, though; but, yes, you make a good point. It's still going to fail the build if @hallipr beats you to setting up the link checker vs. publishing track 2 betas to crates.io. I still think it's worth fixing for that reason.